types/raidgroupconfig.go

```diff
@@ -70,8 +70,8 @@ func (rgc *RaidGroupConfig) Validate() error {
 	case PoolRAIDTypeMirror:
 		{
 			if rgc.GroupDeviceCount != 2 {
-				return errors.Errorf("Invalid device count. Got raid type: %s device count: %d. Expected device count is 2.",
-					rgc.Type, rgc.GroupDeviceCount)
+				return errors.Errorf("Invalid device count %d .for RAID type %q, expected 2",
```

Comment:
@shovanmaity Can we have just one delimiter between statements. I would suggest to have `:`
e.g. 
Invalid device count %d for RAID type %q: Want 2
or
Invalid device count %d: RAID type %q: Want 2

Try to get words that are precise & yet understood.
Avoid abbreviations unless they are well known.

Reply:
This suggestion applies to other messages as well.

Reply:
I have updated ptal once.